tests/functional/auth_tests/test_database_role.py

```diff
@@ -48,12 +49,13 @@ def setup(self, project):
         The existence of this database role triggered the bug as dbt-snowflake attempts
         to revoke it if the user also provides a grants config.
         """
-        role = "BLOCKING_DB_ROLE"
+        unique_suffix = randrange(0, 1_000_000)
+        role = f"BLOCKING_DB_ROLE_{unique_suffix}"
```

Comment:
should we use the invocation_id that's tied to the test as a suffix?

Reply:
We don't have access to the invocation_id, but I can use the test schema prefix.